backend/api/utils/excel_reader_api.py

```diff
@@ -0,0 +1,212 @@
+class Teacher:
+    teacherID = ""
+    teacherFirstName = ""
+    teacherLastName = ""
+    title = ""
+
+
+teacherIDs = []
+teacherID = ""
+teacherIDs.append(teacherID)
+
+teacherFirstNames = []
+teacherFirstName = ""
+teacherFirstNames.append(teacherFirstName)
+
+teacherLastNames = []
+teacherLastName = ""
+teacherLastNames.append(teacherLastName)
+
+titles = []
+title = ""
+titles.append(title)
+
+
+class StudentGroup:
+    groupID = ""
+    groupName = ""
+
+
+groupNames = []
+groupName = ""
+groupNames.append(groupName)
```

Comment:
Bardzo polecam piosenkę: https://www.youtube.com/watch?v=hgI0p1zf31k
a w szczególności jej fragment: "And camelCase is not for Python, never ever."

W Pythonie pisze się:
```python
group_names = ...
```